module_checksum_upgrade/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    'name': 'Module Auto Update Base',
```

Comment:
name and summary does'nt seems to fit with the objective of the module. don't you think ?

Reply:
@legalsylvain I fixed the name. How would you suggest to change the summary?